install/installer/requirements.in

```diff
@@ -0,0 +1,7 @@
+gitpython==3.1.0 # Get info from git
```

Comment:
FYI https://github.com/apps/dependabot-preview can autoupdate in/txt lockfile pair.